cats_and_dogs.md

```diff
@@ -2,26 +2,26 @@
 
 # Cats and dogs
 
-Thank you for taking out time to do this coding challenge. We appreciate it's a significant investment of your time. We respect your time and equally commit to carefully reviewing your solution and bringing it up for debate in your follow-up interview. We have taken a considerable amount of time to design this test to give you a flavour of some of the challenges we have faced in our early days journey. To reduce the complexity and expectation of being versed in medical imaging, we have curated a dataset that simulates some interesting but related challenges in everyday images. 
+Thank you for agreeing to do this coding challenge. We respect your time and equally commit to carefully reviewing your solution and bringing it up for debate in your follow-up interview. We have designed this test to give you a flavour of some of the challenges we have faced in our day-to-day. To reduce the complexity and expectation of being versed in medical imaging, we have curated a dataset that simulates some interesting but related challenges in everyday images. 
```

Comment:
previously we mentioned 'time' 4 sentences in a row

Reply:
**thought:** Removing the word "test" may make it seem less like there is one correct answer.
```suggestion
Thank you for agreeing to do this coding challenge. We respect your time and equally commit to carefully reviewing your solution and bringing it up for debate in your follow-up interview. We have designed this task to give you a flavour of some of the challenges we have faced in our day-to-day. To reduce the complexity and expectation of being versed in medical imaging, we have curated a dataset that simulates some interesting but related challenges in everyday images. 
```